web/react/components/more_direct_channels.jsx

```diff
@@ -47,6 +47,21 @@ export default class MoreDirectChannels extends React.Component {
         UserStore.addChangeListener(this.handleUserChange);
     }
 
+    componentDidUpdate(prevProps) {
+        if (!prevProps.show && this.props.show) {
+            this.onShow();
+        }
+    }
+
+    onShow() {
+        if ($(window).width() > 768) {
```

Comment:
@crspeller Yeah I'm not that concerned about the height, I think it's mainly to remove the perfectscroll, if you're concerned about the height I can do it with css.